backend-mock/index.js

```diff
@@ -359,3 +358,8 @@ app.post("/api/v1/lightning/send-payment", (req, res) => {
     })
   );
 });
+
+app.get("/api/v1/lightning/list-all-tx", (req, res) => {
+  console.log("call to /api/v1/lightning/list-all-tx");
```

Comment:
Wanna switch this to `console.info` to avoid being confused as debugging output and removed?

Reply:
Not really relevant here, since it's just the backend mock.

Aren't `console.info` & `console.log` almost the same?

Reply:
`info` is being displayed differently (blue, info-icon). From my experience `log` is being used for debugging and should be removed before committing. Also eslint will complain about it.

Reply:
I don't really see the benefit in it since it will just be used as a backend server, not running in the browser and only in development.

But I'll changed it anyway since it doesn't hurt. :)

Reply:
No worries, you can also leave it. I'm just worried this eventually will be linted and then eslint will throw a warning/error. `info` indicates this console has been left on pupose.